services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -38,14 +39,21 @@ class PermitAmendment(AuditMixin, Base):
     permit_amendment_status_description = association_proxy('permit_amendment_status',
                                                             'description')
     permit_guid = association_proxy('permit', 'permit_guid')
-    mine_guid = association_proxy('permit', 'mine_guid')
     permit_amendment_type = db.relationship('PermitAmendmentTypeCode')
     permit_amendment_type_description = association_proxy('permit_amendment_type', 'description')
 
     security_total = db.Column(db.Numeric(16, 2))
     now_application_guid = db.Column(
         UUID(as_uuid=True), db.ForeignKey('now_application_identity.now_application_guid'))
     now_identity = db.relationship('NOWApplicationIdentity', lazy='select')
+    mine = db.relationship('Mine', lazy='selectin')
+
+    mine_permit_xref = db.relationship(
+        'MinePermitXref',
+        uselist=False,
+        primaryjoin=
+        "and_(PermitAmendment.mine_guid==foreign(MinePermitXref.mine_guid), PermitAmendment.permit_id==foreign(MinePermitXref.permit_id))"
+    )
```

Comment:
This relationship feels weird to me in that it is only needed for the factories to work.

Reply:
A way of managing this is to allow the permit_amendment factory to create the xref record, therby ensuring that it doesn't exist, i don't it currently works that way, but that would be ideal.

Reply:
added a code comment clarifying.